yarn-project/noir-contracts/src/contracts/rollup_native_asset_contract/src/balance_utils.nr

```diff
@@ -0,0 +1,51 @@
+use dep::aztec3::notes::value_note::Note;
```

Comment:
For my understanding - whats the point of this file? Is all of this just syntactic sugar?

Reply:
Its an implementation to getcompute the balance of a user.